geode-core/src/main/java/org/apache/geode/internal/admin/remote/ClientHealthStats.java

```diff
@@ -91,7 +91,7 @@
   /**
    * Represents stats for a poolName .
    **/
-  private HashMap<String, String> poolStats = new HashMap<String, String>();
+  private HashMap<String, String> poolStats = new HashMap<>();
```

Comment:
It could be. I wasn't trying to get that invasive to use interfaces over implementations. In this case this was an automated change from cleaning up the redundant use of generic type specifiers.